verilog/CST/declaration.cc

```diff
@@ -225,4 +226,58 @@ const verible::SyntaxTreeNode* GetStructOrUnionOrEnumTypeFromDataDeclaration(
   return GetStructOrUnionOrEnumTypeFromInstantiationType(*instantiation_type);
 }
 
+verible::SymbolPtr ReshapeExtendedFunctionCallInDataDeclaration(
+    verible::SymbolPtr* function_call, verible::SymbolPtr* semicolon) {
+  // Append the semicolon to the function call node
+  verible::SymbolCastToNode(**function_call).AppendChild(std::move(*semicolon));
+
+  // Change kInstantiationType to kReferenceCallBase
+  verible::SyntaxTreeNode& instantiation_base(verible::CheckSymbolAsNode(
+      *ABSL_DIE_IF_NULL(
```

Comment:
I think there is no way this would be triggered (at least before changing the second `DIE_IF_NULL`) but just to be safe I have changed the arrangement a little. It will attempt to continue the old method if it is not successful.